src/ert/simulator/batch_simulator_context.py

```diff
@@ -122,7 +113,10 @@ def __post_init__(self) -> None:
         )
         for workflow in ert_config.hooked_workflows[HookRuntime.PRE_SIMULATION]:
             WorkflowRunner(workflow, None, self.ensemble).run_blocking()
-        self._sim_thread = self._run_simulations_simple_step()
+
+        self._sim_task = self._loop.create_task(
```

Comment:
Do we need to keep this  self._loop parameter? If everything is in the same loop and thread, we should be able to create tasks in a usual manner

(and then I think we can purge some stuff from ert.async_utils)

Reply:
Yes, we need to keep the loop parameter for the sake of running scheduler asyncio functions (like `kill_all_jobs`) from sync context.

Reply:
But that should again become not-necessary when/if we have only one loop?

Reply:
hm, not really since not sure what loop you will get when running `asyncio.get_running_loop()` from another thread as everest is still thread based.